controllers/machinehealthcheck_controller.go

```diff
@@ -440,7 +440,7 @@ func (r *MachineHealthCheckReconciler) clusterToMachineHealthCheck(o client.Obje
 	}
 
 	// This list should only contain MachineHealthChecks which belong to the given Cluster
-	requests := []reconcile.Request{}
+	var requests []reconcile.Request
```

Comment:
Let's revert these declarations? It's mostly a style preference (and a small saving in initial memory allocation, which is probably ok in most uses). I'd rather not set a precedent to merge changes to declarations for small gains.

Reply:
Hmm I'm not sure I follow.
I think we can probable agree that ```var requests []reconcile.Request``` is better than ```requests := []reconcile.Request{}``` which is also supported by the official goland [code standards](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices). 
I agree it's a minor change and I can understand the argument against merging a minor change, but since it's merged along with other changes and not all by itself I don't think this is a valid argument (unless we decide the whole PR is too minor to be merged).

Reply:
> I think we can probable agree that var requests []reconcile.Request is better than requests := []reconcile.Request{}

I'm trying to avoid these kind of discussions, which are based mostly on personal preferences. 

> but since it's merged along with other changes and not all by itself I don't think this is a valid argument

No. We make it an effort to keep the pull request focused on one task at hand. If all PRs start bringing in minor style changes as well the noise to signal ratio becomes too high.

> unless we decide the whole PR is too minor to be merged

I'm fine merging the unexporting of the method, and removing the unused parameter. The rest of the changes (minor or not) are personal preferences that I'm not comfortable merging, even if they were standing in a different PR on their own.

Reply:
I guess we'll  have to agree to disagree on that.
But since I'm a new guest to this repo I think it's only common curtsy to respect the house rules :slightly_smiling_face: